modules/logging-loki-alerts.adoc

```diff
@@ -33,24 +33,79 @@ The `AlertingRule` CR contains a set of specifications and webhook validation de
 
 .Procedure
 
-1. Create an AlertingRule CR:
-
---
-include::snippets/logging-create-apply-cr-snip.adoc[lines=9..12]
---
-
-2. Populate your AlertingRule CR using the appropriate example below:
-
---
-include::snippets/logging-alertingrule-inf-callouts-snip.adoc[]
---
-
---
-include::snippets/logging-alertingrule-app-callouts-snip.adoc[]
---
-
-3. Apply the CR.
-
---
-include::snippets/logging-create-apply-cr-snip.adoc[lines=14..17]
---
+. Create an `AlertingRule` custom resource (CR):
++
+.Example infrastructure AlertingRule CR
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: AlertingRule
+  metadata:
+    name: loki-operator-alerts
+    namespace: openshift-operators-redhat <1>
+    labels: <2>
+      openshift.io/cluster-monitoring: "true"
+  spec:
+    tenantID: "infrastructure" <3>
+    groups:
+      - name: LokiOperatorHighReconciliationError
+        rules:
+          - alert: HighPercentageError
+            expr: | <4>
+              sum(rate({kubernetes_namespace_name="openshift-operators-redhat", kubernetes_pod_name=~"loki-operator-controller-manager.*"} |= "error" [1m])) by (job)
+                /
+              sum(rate({kubernetes_namespace_name="openshift-operators-redhat", kubernetes_pod_name=~"loki-operator-controller-manager.*"}[1m])) by (job)
+                > 0.01
+            for: 10s
+            labels:
+              severity: critical <5>
+            annotations:
+              summary: High Loki Operator Reconciliation Errors <6>
+              description: High Loki Operator Reconciliation Errors <7>
+----
+<1> The namespace where this `AlertingRule` CR is created must have a label matching the LokiStack `spec.rules.namespaceSelector` definition.
+<2> The `labels` block must match the LokiStack `spec.rules.selector` definition.
+<3> `AlertingRule` CRs for `infrastructure` tenants are only supported in the `openshift-\*`, `kube-\*`, or `default` namespaces.
+<4> Value for `kubernetes_namespace_name:` must match the value for `metadata.namespace`.
```

Comment:
```suggestion
<4> The value for `kubernetes_namespace_name:` must match the value for `metadata.namespace`.
```
Per [ISG guidance on list items](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#wording-of-list-items), all of them must be grammatically parallel. So all of these need to be complete sentences.